frontend/src/core/editor/hooks/useHandleShortcuts.js

```diff
@@ -148,7 +148,7 @@ export default function useHandleShortcuts(): ((
             }
 
             const numTranslations =
-                translations.length + (searchResults && searchResults.length);
+                translations.length + (searchResults?.length || 0);
```

Comment:
I love optional chaining!

We'll need to update our [browser support matrix](https://github.com/mozilla/pontoon/blob/master/CONTRIBUTING.rst#browser-support) to start [using](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility) it, though. Which is probably out of date, anyways.

Given Firefox ESR supports it, I'm fine with the change.

Reply:
Yay!!!

Reply:
tl;dr: chaining works fine w/out any action required.

I've spent a few hours to find out what's going on here. `react-scripts` hard-code transpiling optional chaining away, due to webpack due to babel due to acorn. Or so. There's tons of major updates out there in our frontend build stack.